testcafe/runner-drive.js

```diff
@@ -5,22 +5,26 @@ async function runRunner() {
   const runner = await tc.createRunner()
   const response = await runner
     .src([
+      //Init data : Unzip archive with files to upload
+      'testcafe/tests/helpers/init-data.js',
+
+      //Tests !
       'testcafe/tests/drive/drive_nav.js',
       'testcafe/tests/drive/folder_sharing_scenario.js',
-      'testcafe/tests/drive/file_sharing_scenario.js'
+      'testcafe/tests/drive/file_sharing_scenario.js',
+      'testcafe/tests/drive/drive-viewer-feature.js'
     ])
     //emulation:cdpPort=9222 is used to set the download folder in headless mode
     .browsers(['chrome:headless:emulation:cdpPort=9222 --start-maximized'])
 
     .screenshots(
-      'reports/screenshots/',
+      'reports/',
       true,
-      '${DATE}_${TIME}/test-${TEST_INDEX}-${FILE_INDEX}.png'
+      '${DATE}_${TIME}/test-${TEST}-${FILE_INDEX}.png'
```

Comment:
they are path pattern from testcafe : https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/screenshots-and-videos.html#path-pattern-placeholders
The screenshot path might change soon anyway for visual testing needs

Reply:
Ah yes, thank you. I'm surprised that the linter doesn't complain about these variables though? Normally global variables like that are declared in a comment at the top f the file, like [here](https://github.com/cozy/cozy-drive/blob/master/src/drive/targets/mobile/InitAppMobile.jsx#L1).

Reply:
Maybe because they are strings : I use `'` not ``` (as testcafe tels me to do! https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#screenshots

Reply:
Aaaah yes of course I missed that! Thanks 👍